docs/data-sources/scalr_workspace.md

```diff
@@ -36,6 +36,7 @@ All arguments plus:
 * `vcs_repo` - If a workspace is linked to a VCS repository this block shows the details, otherwise `{}`
 * `created_by` - Details of the user that created the workspace.
 * `has_resources` - The presence of active terraform resources in the current state version.
+* `auto_queue_runs` - Boolean indicates if runs should be queued by VCS changes.
```

Comment:
Let's consolidate descriptions for `auto_queue_runs`, so that they are the same.